### DISCUSSION TITLE
Feature request: Grid Plots in Foxglove Studio

### DISCUSSION THREAD

**Discussion by Pathorse** [foxglove/community#852] [Q&A]
### Feature Description

Grid plots in Foxglove Studio as illustrated below:

![image_720](https://github.com/foxglove/community/assets/35997875/539c89f5-e286-47fd-9508-75ec017782c2)

With this feature it would also be great to have the option to set axis limits and settings for entire grids rather than 1 and 1 plot.

### Use Cases
There are a lot of relevant use cases for this within robotics, since the feature to do a larger number of plots is rather general, but a few notable can be:
- Sensor Fusion Analysis:
- Multi-joint Analysis for Manipulator Arms:
- Motion Planning and Path Analysis:
- Comparative Analysis of Different Algorithms:
- Tuning of Control Parameters:
- Learning and Adaptation Analysis:
- Communication Analysis:
- Environmental Analysis:

**Comment by amacneil** [comment]
This is a good idea (not currently on our roadmap).

For future reference, I'm recording an idea that someone internally suggested: having a UI similar to Google Docs when you create a table:

![image](https://github.com/foxglove/community/assets/637671/63726616-efdc-4c4e-9fdb-786c14174c0f)

So basically allow me to create a grid of panels, without having to manually resize them.

That wouldn't necessarily solve "it would also be great to have the option to set axis limits and settings for entire grids", we would need to think about that separately.
